### PR TITLE
Dyn 5358 user friendly splash screen

### PR DESCRIPTION
**Pull request by filipeotero**
This PR includes the following improvements for a more user friendly splash screen:

- Make splash screen 20% bigger
- Add close button
- Add hover interaction in buttons
- Show total loading time in static page
- Clicking on import settings, the file picker should filter out only xml files
- Dynamo triggered with Enter key

friendlySplashScreen

FYI: @QilongTang @avidit

### CODE REVIEW

File: src/Static.js [hunk 1]
```diff
@@ -28,11 +26,18 @@ class Static extends React.Component {
       errorDescription: 'Something went wrong when importing your custom setting file. Please try again or proceed with default settings.',
       signInTitle: this.props.signInTitle,
       signInStatus: this.props.signInStatus,
-      importSettingsTitle: this.props.importSettingsTitle,
+      loadingTime: 'Total loading time: ',
```
Comment (QilongTang) [inline src/Static.js:29]: Do we need to use `this.props.` here or?

**Review by QilongTang — APPROVED** [review]
LGTM with one comment. Also please update the package version to 1.0.0

File: src/Static.js [hunk 2]
```diff
@@ -103,29 +115,29 @@ class Static extends React.Component {
...
-        this.setState({ signInTitle: "Signing In" })
-        var ret = await chrome.webview.hostObjects.scriptObject.SignIn();
-        this.setState({ signInStatus: ret });
+        this.setState({ signInTitle: 'Signing In' });
+        var ret2 = await chrome.webview.hostObjects.scriptObject.SignIn();
+        this.setState({ signInStatus: ret2 });
 
         btn.classList.remove('disableButton');
         btn.disabled = false;
         if (ret) {
```
Comment (avidit) [inline src/Static.js:132]: Does `ret` needs to be renamed to `ret2` here as well?

File: src/Static.js (hunk 1 above)
Comment (filipeotero) [reply src/Static.js:29]: No need for that since we are using `this.state`

File: src/Static.js (hunk 2 above)
Comment (filipeotero) [reply src/Static.js:132]: I've changed the name because eslint was giving errors. It was changed to `status`, a better term. Thanks!

**Pull request merged**